tool/gravity/cli/operation.go

```diff
@@ -393,6 +361,24 @@ func (r *backendOperations) listInstallOperation() error {
 	return trace.NotFound("no operation found")
 }
 
+func (r *backendOperations) updateOperationInCache(getter operationGetter, logger logrus.FieldLogger) {
```

Comment:
Probably not for this PR, but I really think we should try to move all operation handling/displaying logic somewhere to lib/ and cover with tests. It's become pretty non-trivial logic.

Reply:
I already attempted it at some point but abandoned due to a non-trivial amount of work. I agree though and will reattempt in a later PR.